pip/_vendor/pkg_resources/__init__.py

```diff
@@ -2941,10 +2941,10 @@ def _get_mro(cls):
     """Get an mro for a type or classic class"""
     if not isinstance(cls, type):
 
-        class cls(cls, object):
+        class new_cls(cls, object):
             pass
 
-        return cls.__mro__[1:]
+        return new_cls.__mro__[1:]
```

Comment:
This bothers me, because we don't normally modify vendored files. But I guess it's needed to stop mypy complaining about the pkg_resources code.